rllib/agents/sac/sac.py

```diff
@@ -46,13 +46,14 @@
     "evaluation_num_episodes": 1,
     # Extra configuration that disables exploration.
     "evaluation_config": {
-        "exploration_enabled": False,
+        "model": {
+            "deterministic_action_sampling": True,
```

Comment:
Seems like this should be part of exploration API right?

Reply:
Deterministic action sampling (from the model's distribution) is handled separately from Explorations. That's why "soft-q" is more like a deterministic/stochastic sampling setting. Less an Exploration component.
The difference is that Exploration is (almost always) switched off during evaluation/inference/post-deployment, where action sampling (stochastically) is not.

Reply:
Why should it? The softmax for PPO discrete is going to be part of exploration there right?